go/discord-command-bot/cli/setup.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func main() {
+	err := errorIfEnvMissing([]string{
+		"DISCORD_PUBLIC_KEY",
+		"DISCORD_APPLICATION_ID",
+		"DISCORD_TOKEN",
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	registerApi := "https://discord.com/api/v9/applications/" + os.Getenv("DISCORD_APPLICATION_ID") + "/commands"
+
+	bodyJson := map[string]string{"name": "hello", "description": "Hello World Command"}
+	bodyString, err := json.Marshal(bodyJson)
```

Comment:
we might wanna create a struct for this with field tags 👍🏻

Reply:
Its overkill here I would say